index.html

```diff
@@ -2307,6 +2992,350 @@ <h2>
+          </dd>
+          <dt>
+            <dfn>postalCode</dfn> member
+          </dt>
+          <dd>
+            Denotes that the <a>postal code</a> has a validation error. In the
+            user agent's UI, this member corresponds to the input field that
+            provided the {{ContactAddress}}'s {{ContactAddress/postalCode}}
+            attribute's value.
+          </dd>
+          <dt>
+            <dfn>recipient</dfn> member
+          </dt>
+          <dd>
+            Denotes that the <a>recipient</a> has a validation error. In the
+            user agent's UI, this member corresponds to the input field that
+            provided the {{ContactAddress}}'s {{ContactAddress/addressLine}}
+            attribute's value.
+          </dd>
+          <dt>
+            <dfn>region</dfn> member
+          </dt>
+          <dd>
+            Denotes that the <a>region</a> has a validation error. In the user
+            agent's UI, this member corresponds to the input field that
+            provided the {{ContactAddress}}'s {{ContactAddress/region}}
+            attribute's value.
+          </dd>
+          <dt>
+            <dfn>sortingCode</dfn> member
+          </dt>
+          <dd>
+            The <a>sorting code</a> has a validation error. In the user agent's
+            UI, this member corresponds to the input field that provided the
+            {{ContactAddress}}'s {{ContactAddress/sortingCode}} attribute's
+            value.
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h2>
+          Creating a `ContactAddress` from user-provided input
+        </h2>
+        <p>
+          The steps to <dfn data-export="">create a `ContactAddress` from
+          user-provided input</dfn> are given by the following algorithm. The
+          algorithm takes a <a>list</a> |redactList|.
+        </p>
+        <div class="note" title=
+        "Privacy of recipient information (the redactList)">
+          <p>
+            The |redactList| optionally gives user agents the possibility to
+            limit the amount of personal information about the recipient that
+            the API shares with the merchant.
+          </p>
+          <p>
+            For merchants, the resulting {{ContactAddress}} object provides
+            enough information to, for example, calculate shipping costs, but,
+            in most cases, not enough information to physically locate and
+            uniquely identify the recipient.
+          </p>
+          <p>
+            Unfortunately, even with the |redactList|, recipient anonymity
+            cannot be assured. This is because in some countries postal codes
+            are so fine-grained that they can uniquely identify a recipient.
+          </p>
+        </div>
+        <ol>
+          <li>Let |details:AddressInit| be a newly created {{AddressInit}}
+          dictionary.
+          </li>
+          <li>If "addressLine" is not in |redactList|, set
```

Comment:
Make this into a switch statement off each redactList value, and pop as you go... weeeee